### DISCUSSION TITLE
Mickey Mouse watchface

### DISCUSSION THREAD

**Discussion by Baggypants** [InfiniTimeOrg/InfiniTime#1931] [Ideas]
The Mickey Mouse watch has been available in various forms since [1933](https://blog.watchdoctor.biz/2017/06/05/history-of-the-mickey-mouse-watch/) and has also passed into public conciousness as a shorthand for cheap items pandering to consumerism. From January 1st 2024 the copyright over the short film "Steamboat Willie" expires and the design of Mickey Mouse used in that film becomes [Public Domain](https://pluralistic.net/2023/12/15/mouse-liberation-front/).

I'd like to propose the Micky Mouse Watchface, using the design in Steamboat Willie, as being a design that will stand for openness, creative freedom and promote the legal use of previously
copyright works.

Based on the existing Analogue watchface existing in the software already.

**Comment by 88572** [comment]
While the design for Mickey Mouse may be in the public domain, the watch may not be. 
_(I'm not a lawyer, don't cite me on this)_

**Comment by Baggypants** [comment]
The image of steamboat willie mickey will be public domain.
The images found on the extant mickey mouse watches will not be public domain. Those would not be used.
The idea of putting an image on a watch wouldn't be covered by copyright, or owned by Disney.

**Reply by 88572** [reply]
Alright, that makes sense.

**Comment by riban-bw** [comment]
Maybe you could extract the relevant images.
